website/addons/wiki/templates/log_templates.mako

```diff
@@ -17,3 +17,27 @@ deleted wiki page
 <a data-bind="attr: {href: wikiUrl}, text: params.page"></a>
 of <a class = "log-node-title-link overflow" data-bind="text:nodeTitle, attr: {href: nodeUrl}"></a>
 </script>
+
+<script type="text/html" id="made_wiki_public">
+made the wiki
+of <a class = "log-node-title-link overflow" data-bind="text:nodeTitle, attr: {href: nodeUrl}"></a>
+publicly editable
+</script>
+
+<script type="text/html" id="made_wiki_public_no_user">
```

Comment:
In what context can a wiki be made publicly editable without a user requesting to do so?